doc/guide/guide-basics.rst

```diff
@@ -114,7 +114,9 @@ We can create a ``Qobj`` with a user defined data set by passing a list or array
      [0.60111501 0.70807258 0.02058449 0.96990985]
      [0.83244264 0.21233911 0.18182497 0.18340451]]
 
-Notice how both the dims and shape change according to the input data.  Although dims and shape appear to have the same function, the difference will become quite clear in the section on :ref:`tensor products and partial traces <tensor>`.
+Notice how both the dims and shape change according to the input data.  Although dims and shape appear to be the same, 
+dims keep track of the shapes for individual components of a multipartite system, while shape does not. See the section 
+on :ref:`tensor products and partial traces <tensor>`.
```

Comment:
```suggestion
Notice how both ``dims`` and ``shape`` change according to the input data.  Although ``dims`` and ``shape`` appear to be the same, 
``dims`` keeps track of the shapes for individual components of a multipartite system, while ``shape`` does not. We refer the reader to  section :ref:`tensor products and partial traces <tensor>` for more information.
```
I surrounded attributes with `` but im not sure that is the preferred way. What do you think @hodgestar ?